src/Commands/CleanCommand.php

```diff
@@ -22,7 +22,9 @@ class CleanCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'medialibrary:clean {modelType?} {collectionName?} {disk?} {--dry-run}';
+    protected $signature = 'medialibrary:clean {modelType?} {collectionName?} {disk?} 
+    {--dry-run : List files that will be removed without removing them},
+    {-- force : Force the compiled class file to be written}';
```

Comment:
The description of `--force` seems to have been copy pasted 😄 
Could you improve the description?